Tests/js/views/assets/ez-userprofileview-tests.js

```diff
@@ -109,6 +110,49 @@ YUI.add('ez-userprofileview-tests', function (Y) {
             Y.Assert.areSame(this.view, item.getTargets()[0], 'Should set correct event target');
             Y.Assert.isInstanceOf(Y.eZ.UserMenuView, item, "The user menu view should contain an instance of the constructor");
         },
+
+    });
+
+    userMenuDisplayTest = new Y.Test.Case({
+        name: "eZ User Profile attribute test",
+
+        setUp: function () {
+            var View = Y.Base.create('userMenuView', Y.View, [], {}, {
+                    ATTRS: {
+                        displayed: {
+                            value: false,
+                        }
+                    }
+                });
+            this.view = new Y.eZ.UserProfileView({
+                userMenuView: new View(),
+            });
+            this.view.get('userMenuView').addTarget(this.view);
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should add the menu displayed class": function () {
+            this.view.get('userMenuView').set('displayed', true);
+
+            Assert.isTrue(
+                this.view.get('container').hasClass('is-menu-displayed'),
```

Comment:
done but that's highly debatable :)
if you see this class as just a string and an implementation detail then yes you are perfectly right.
But in this case (ie a test), this class is like a public method (or a public behaviour), this is something that as of the release should stay for BC reason. For instance, you wouldn't put `'container'` or `'get'` (the method name) in variables (even if this test uses a lot those strings).
(in the implementation, we use variables because it can improve compression and minification efficiency and also because it improves the readability in that context)